Exiled.Installer/Program.cs

```diff
@@ -280,20 +279,27 @@ static PathResolution TryParse(string s)
         private static Release FindRelease(CommandSettings args, IEnumerable<Release> releases)
         {
             Console.WriteLine(Resources.Program_TryFindRelease_Trying_to_find_release__);
-            Version targetVersion = args.TargetVersion is not null ? new Version(args.TargetVersion) : new Version(releases.First().TagName);
+            Version? targetVersion = args.TargetVersion is not null ? new Version(args.TargetVersion) : null;
 
-            foreach (Release r in releases)
-            {
-                if (targetVersion != new Version(r.TagName))
-                    continue;
+            var enumerable = releases.ToList();
 
-                if (targetVersion.IsPreRelease && !args.PreReleases)
-                    continue;
+            foreach (var release in enumerable)
```

Comment:
Do not use `var`.

Reply:
Oh yeah mb. I think it might be a good idea to prevent their use by adding an option in .editorconfig

Reply:
Done